<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessonSelection</class>
 <widget class="QWidget" name="SessonSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>669</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1440</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>841</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#table_bh{
	background-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Tìm kiếm theo:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_timkiem">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Mã môn học</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tên môn học</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Giảng viên</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lớp</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="txt_timkiem">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_timkiem">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tìm kiếm</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_bh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>821</width>
      <height>521</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <column>
     <property name="text">
      <string>Mã buổi học</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tên môn học</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tuần</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Giảng viên</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tiết bắt đầu</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tiết kết thúc</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ngày</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Lớp</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_accept">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>620</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Chọn buổi học</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
